docs/build/identity.rs/1.1/docs/faq.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: Frequently Asked Questions
+sidebar_label: FAQ
+description: Frequently Asked Question regarding IOTA Identity.
+image: /img/Identity_icon.png
+keywords:
+  - FAQ
+  - Frequently Asked Question
+  - Troubleshooting
+  - IOTA Identity
+---
+
+This page contains frequently asked questions regarding the Identity Library and Self Sovereign Identity in general.
+
+### What programming languages are supported by the IOTA Identity framework?
+
+We currently provide a Rust library and a JavaScript library for both the browser and Node.js via WebAssembly (Wasm) bindings. See the "Programming Languages" section for more information.
+
+### Do I need to have IOTA tokens to start building with IOTA Identity?
```

Comment:
```suggestion
### Do I need IOTA tokens to start building with IOTA Identity?
```

Reply:
Testenet aside, don't they need some tokens to cover storage deposits?

Reply:
you are right, this is outdated

Reply:
Made a quick update and also changed the permanode FAQ